### PR TITLE
Prepare to release v6.0.0

### LINKED ISSUE #448: Release 6.0.0
Depends on:
  - [x] https://github.com/Polymer/wct-local/issues/50
  - [x] https://github.com/Polymer/web-component-tester/pull/474

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

Fixes #448

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,19 @@ All notable changes to this project will be documented in this file.
...
+    * As part of this, wct will also require that the project under test have an installation of the client side web-component-tester bower package.
+    * This release unifies the behavior of `wct` and `polyserve`, so if your code works without warnings in one it should work in the other.
+  * `webserver.webRunnerPath`, `webserver.webRunnerContent`, and `webserver.urlPrefix`, `webserver.staticContent` were internal properties that were exposed on the `config` object. They have been refactored and their replacement has been prefixed with an underscore to clarify that they're internal implementation details.
+  * Dropped support for node v4, added support for node v8. See our [node version support policy](https://www.polymer-project.org/2.0/docs/tools/node-support) for details.
+
+<details>
+  <summary>Click for full details</summary>
```
Comment (azakus) [inline CHANGELOG.md]: This seems weird

File: CHANGELOG.md [hunk 2]
```diff
@@ -49,6 +61,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * Fixed #373 and #383 which were caused by `emitHook` not handling arguments correctly.
 * Fixed error log message for loading WCT config
 
+</details>
```
Comment (azakus) [inline CHANGELOG.md]: Remove?

**Comment by rictic** [comment]
:D

**Comment by rictic** [comment]
:D

**Comment by notwaldorf** [comment]
:shipit:

**Pull request merged**